### ISSUE TITLE
Update marble diagram images for operators that accept "notifiers"

### ISSUE THREAD

**Issue by jakovljevic-mladen** [ReactiveX/rxjs#6627]
Since RxJS v7 has been released, many operators that accept "notifiers" have been adjusted to listen to only "next" notifications. This has become mandatory as per [RxJS Core Semantics](https://github.com/ReactiveX/rxjs/blob/7dd7f3c4ab/docs_app/content/guide/core-semantics.md) guide:

> Operators that accept "notifiers" (as described above), MUST ONLY recognized next values from the notifier as "notifications". Emitted completions may not be used a source of notification.

However, many marble diagram images have to be updated as they don't reliably picture this behavior. For example, `audit` operator displays this image:
![audit](https://rxjs.dev/assets/images/marble-diagrams/audit.png)

A user "reading" this image might be deceived to think that an Observable that has been passed to `audit` operator needs to complete before destination will receive next "next" notification. This is not true now.

Marble diagram images of these operators need to be updated:
- [x] [audit](https://rxjs.dev/api/operators/audit) (fixed in PR: #6636)
- [x] [debounce](https://rxjs.dev/api/operators/debounce) (fixed in PR: #6639)
- [x] [throttle](https://rxjs.dev/api/operators/throttle) (fixed in PR: #6638)
- [x] [bufferWhen](https://rxjs.dev/api/operators/bufferWhen) (fixed in PR: #6767)
- [ ] [windowWhen](https://rxjs.dev/api/operators/windowWhen) (PR: #7052)

**Comment by niklas-wortmann** [comment]
We introduced swirly a while back to generate svg marble diagrams. I did not have the time to update/add all the marble diagrams, but I did a poc with concatAll that is already deployed to rxjs.dev, you basically describe the marble diagrams in a ascii format, so there should be a concatAll.txt somewhere in the repo, additionally there are dedicated npm scripts to use those text files and generate the marble diagram. Lastly, you would just have to update the path in the tsdoc comment of the operator, likely just replacing the fileformat `png` with `svg`

**Comment by jakovljevic-mladen** [comment]
@benlesh, since you were mainly involved in normalizing behavior between various operators that receive notifying Observables so that they all "act" on _next_ notification, could you please review the list I created and possibly give suggestions?

There are a lot more operators that receive notifying Observables than what's on the list, but it looks like they have relatively good marble diagrams, so I didn't add them here.

**Comment by ladyleet** [comment]
@niklas-wortmann @benlesh following up on this one!

**Comment by jakovljevic-mladen** [comment]
@benlesh, I was once again reviewing this issue and I'm still not sure whether `windowWhen` marble diagram should be fixed. What it looks like, a `windowWhen` [test](https://github.com/ReactiveX/rxjs/blob/3330ff3d861d4342190c7c7a092fdd9a3c234c05/spec/operators/windowWhen-spec.ts#L17) suggests that it also accepts complete notifications. This is also suggested [here](https://github.com/ReactiveX/rxjs/blob/888c753e272443bbb879f9ffb1a8145a7773c865/src/internal/operators/windowWhen.ts#L98) where `OperatorSubscriber` accepts third parameter which is the same as the second one (the second `openWindow`) - the third parameter is a function that gets called on complete notifications, which indicates that the same thing will happen on both next and complete events.

In contrary, `bufferWhen` [seems to accept](https://github.com/ReactiveX/rxjs/blob/888c753e272443bbb879f9ffb1a8145a7773c865/src/internal/operators/bufferWhen.ts#L69) `noop` as the third parameter to `OperatorSubscriber`, which suggests that `bufferWhen` does not accept complete notifications. Is this a bug or this behavior seems like an exception for some reason? If it is a bug, please let me know, I can create a separate ticket for that.
